docker_config/compose_files/docker-compose.dev.yml

```diff
@@ -11,6 +11,8 @@ services:
     stdin_open: true
     tty: true
   rest-server:
+    stdin_open: true
```

Comment:
Note: I added these entries in the dev Docker compose so I could use pdb to debug code running in the containers locally. I thought others might want to use pdb in the future as well, so I've left this in. However, I can take it out if that's preferable!

Reply:
Done.